src/main/java/net/dries007/tfc/common/blocks/ExtendedProperties.java

```diff
@@ -141,9 +142,9 @@ public ExtendedProperties pathType(BlockPathTypes type)
         return this;
     }
 
-    public ExtendedProperties enchantmentPower(int power)
+    public ExtendedProperties enchantmentPower(ToFloatFunction<BlockState> powerGetter)
```

Comment:
Can you add an overload for this this that takes a single `int` (for backwards binary compat)